templates/pages/setup/notification/mailing_setting.html.twig

```diff
@@ -356,19 +356,72 @@
-    </form>
-{% else %}
-    <div class="row">
-        <div class="col">
-            <div class="alert alert-info">
-                <i class="fas fa-info-circle fa-2x alert-icon"></i>
-                {{ __('Notifications are disabled.') }}
-                <a href="{{ config('root_doc') }}/front/setup.notification.php">{{ __('See configuration') }}</a>
+
+            <!-- Modal -->
+            <div class="modal fade" id="mailTestLogModal" tabindex="-1" role="dialog" aria-labelledby="mailTestLogModalLabel" aria-hidden="true">
+                <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title" id="mailTestLogModalLabel">{{ __('Mail Test Logs') }}</h5>
+                            <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
+                        </div>
+                        <div class="modal-body">
+                            <div>
+                                <h4>Result</h4>
+                                <span class="badge mb-3">{{ __('Success') }}</span>
+                            </div>
+                            <div>
+                                <h4>Error</h4>
+                                <pre style="white-space: pre-wrap;"></pre>
+                            </div>
+                            <div>
+                                <h4>Logs</h4>
+                                <pre></pre>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <script>
+                    $('#test-email-btn').on('click', function () {
+                        const $modal = $('#mailTestLogModal');
+
+                        $.ajax({
+                            url: CFG_GLPI.root_doc + '/ajax/notificationmailingsettings.php',
+                            method: 'POST',
+                            datatype: 'json',
+                            data: {
+                                'test_smtp_send': 1
+                            }
+                        }).done(function (data) {
+                            $modal.find('.modal-body div:nth-child(1) span').text(data.success ? '{{ __('Success') }}' : '{{ __('Error') }}').toggleClass('bg-success', data.success).toggleClass('bg-danger', ! data.success);
+                            $modal.find('.modal-body div:nth-child(2) pre').text(data.error);
+                            $modal.find('.modal-body div:nth-child(3) pre').text(data.debug);
+
+                            if (data.error == null || data.error == '') {
+                                $modal.find('.modal-body div:nth-child(2)').hide();
+                            }
+                            if (data.debug == null || data.debug == '') {
+                                $modal.find('.modal-body div:nth-child(3)').hide();
+                            }
```

Comment:
It would probably be easier to build the HTML directly using a javascript template literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).